tools/highlight-files/BUILD.bazel

```diff
@@ -21,5 +22,10 @@ nodejs_binary(
         "@npm//fs-extra",
         "@npm//highlight.js",
     ],
+    templated_args = [
+            # TODO(josephperrott): update dependency usages to no longer need bazel patch module resolver
+            # See: https://github.com/bazelbuild/rules_nodejs/wiki#--bazel_patch_module_resolver-now-defaults-to-false-2324
```

Comment:
Is it intentional to only add this comment for a couple of instances of this flag?